apps/src/templates/progress/ProgressLegend.jsx

```diff
@@ -51,6 +38,13 @@ class ProgressLegend extends Component {
           </tr>
           <tr style={styles.secondRow}>
             <TD colSpan={4} />
+            {this.props.includeProgressNotApplicable && (
+              <TD>
```

Comment:
Why is `TD` in caps?

Reply:
Oh, I see `TD` is a custom component, that seems very unexpected. Could we just define a style constant and pass to all `td`s? Or give it a different name?

Reply:
this whole component could really use a refactor, along with `SummaryViewLegend`. i thought long and hard about doing that as part of this work, but ultimately opted for a surgical update. i would prefer to defer cleaning up that `TD` stuff for a more comprehensive refactor.